src/gen/opcodes.rs

```diff
@@ -33,6 +33,9 @@ pub const ASSERT_SECONDS_ABSOLUTE: ConditionOpcode = 81;
 pub const ASSERT_HEIGHT_RELATIVE: ConditionOpcode = 82;
 pub const ASSERT_HEIGHT_ABSOLUTE: ConditionOpcode = 83;
 
+// no-op condition
+pub const ALWAYS_TRUE: ConditionOpcode = 90;
```

Comment:
How about using `0`?

Reply:
Or if that's not possible, use `1` (so it looks like `q`).

Reply:
sure. that's a single byte string containing `0`, right? as all opcodes are currently one byte.

Reply:
I was actually thinking the `0` number, ie. null, the empty string. If you're using a "first byte" decoding rather than a "convert to clvm int" mechanism, we can use `1` (ie. the single byte `0x01`).